packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_pricing_units.py

```diff
@@ -0,0 +1,64 @@
+""" Pricing details table
+    - each pricing plan table can have multiple units. These units are stored in the
+    pricing details table with their unit cost. Each unit cost (row in this table) has
+    id which uniquely defines the prices at this point of the time. We always store whole
+    history and do not update the rows of this table.
+"""
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB
+
+from ._common import column_created_datetime, column_modified_datetime
+from .base import metadata
+
+resource_tracker_pricing_units = sa.Table(
+    "resource_tracker_pricing_units",
+    metadata,
+    sa.Column(
+        "pricing_unit_id",
+        sa.BigInteger,
+        nullable=False,
+        primary_key=True,
+        doc="Identifier index",
+    ),
+    sa.Column(
+        "pricing_plan_id",
+        sa.BigInteger,
+        sa.ForeignKey(
+            "resource_tracker_pricing_plans.pricing_plan_id",
+            name="fk_resource_tracker_pricing_units_pricing_plan_id",
+            onupdate="CASCADE",
+            ondelete="CASCADE",
```

Comment:
Q: in case of deletion of the key from the likened table this entry will also be removed (if I understood this correctly). Please check that it is like this and that you what this behaviour.

Reply:
Yes correct that is the behaviour I want. If pricing-plan is deleted, all of the pricing-units connected to that pricing-plan should be deleted. NOTE: in the pricing-unit-costs I store all history and never delete it, therefore thats the table with all historical information :+1: